### ISSUE TITLE
Add option to disable block previews in the inserter.

### ISSUE THREAD

**Issue by La-Geek** [WordPress/gutenberg#22053]
**Describe the bug**
In Gutenberg 8.0 the option to uncheck "Inserter Help Panel" is not available anymore.

**To reproduce**
Gutenberg 8.0 = Navigate -> "More tools & options -> Options

**Expected behavior**
There should be the checkbox "Inserter Help Panel"

**Screenshots**

### **WordPress 5.4.1:**
![wp_5_4_1__options_inserter_help_panle](https://user-images.githubusercontent.com/1585416/80912214-b192db80-8d3b-11ea-9a23-3522001e7245.jpg)

### **Gutenberg 8.0:**
![gutenberg_8_missing_inserter_help_panel](https://user-images.githubusercontent.com/1585416/80912186-860ff100-8d3b-11ea-9111-d4fac9a29342.jpg)

**Comment by talldan** [comment]
@La-Geek The help panel was also removed, so the option to enable/disable them is no longer relevant.

**Comment by La-Geek** [comment]
Yes, but instead there is the welcome guide. It is enough, if I see it once, but not every time (after I have cleared caches and cookies)? Do you have a snippet for the functions.php?

**Comment by talldan** [comment]
It looks like @La-Geek is referring to are the Block Previews in the inserter:
<img width="675" alt="Screenshot 2020-05-04 at 6 10 25 pm" src="https://user-images.githubusercontent.com/677833/80955848-90f97d00-8e32-11ea-805f-3ba55e4bd660.png">

Which replaced what used to be the inserter help panel:
![Screen Shot 2020-01-16 at 8 32 24 AM](https://user-images.githubusercontent.com/617986/72543685-f5201300-383a-11ea-9290-62c851c5a27f.png)

Reopening this issue as an option to disable the previews might be useful for some users.

**Comment by maddisondesigns** [comment]
Just like the previous tips, and Inserter Help Panel, the Block previews are mostly useless and take up valuable screen real-estate by covering content. Once you're familiar with the editor, it's unlikely that you'll want to see a preview every time you hover over a block. There needs to be a way to way to turn them off permanently in the UI, just like there was for the previous Inserter Help Panel.

**Comment by maddisondesigns** [comment]
When are going to be able to turn off these Block previews? It's frustrating having your content covered up unnecessarily and the constant flickering as you hover over each block is distracting and annoying.

**Comment by ovidiul** [comment]
Seems the setting controlling the inserter blocks preview is always set to true [here](https://github.com/WordPress/gutenberg/blob/17e3641efa0ca29f0d899555487d337222871c20/packages/edit-post/src/components/secondary-sidebar/inserter-sidebar.js#L50), maybe we can add a setting in the editor to toggle this?

**Comment by daveyjones** [comment]
I agree that block previews should go away completely, since their usefulness is questionable and it adds one more hurdle to get over when creating a new block. But for now, I've just added this to my CSS:

```
.block-editor-inserter__preview-container {
  display: none;
}
```

**Comment by youknowriad** [comment]
I'm closing this as not planned. In the meantime the CSS hack above should be fine for most cases. We can reconsider later if needed.
